Test fails, can't find @bumaga/tabs

If I use a new Create React App, then use this library, the CRA Jest test fails:
`Cannot find module '@bumaga/tabs' from 'SomeComponent.js'`

If I change the import to a more explicit:
`import { Tabs, useTabState, Panel } from "@bumaga/tabs/dis/index";`
It also fails, but with a different error.

What could this be?

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "@bumaga/tabs",
   "version": "0.1.2",
-  "main": "dist/index.js",
+  "main": "dist/index.cjs.js",
   "module": "dist/index.js",
   "files": [
     "dist"

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
   ],
   "repository": "jeetiss/tabs",
   "scripts": {
-    "build": "babel src -d dist",
+    "build": "rollup -c",
     "size": "size-limit",
     "prepublishOnly": "yarn build",
     "test": "NODE_ENV=test jest --verbose --coverage"
@@ -39,6 +39,8 @@
     "jest": "^24.8.0",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
+    "rollup": "^1.18.0",
+    "rollup-plugin-babel": "^4.3.3",
     "size-limit": "^1.3.8"
   },
   "size-limit": [

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,0 +1,24 @@
+import babel from 'rollup-plugin-babel'
+
+export default [
+  {
+    input: 'src/index.js',
+    external: ['react', 'use-constant'],
+    output: {
+      format: 'esm',
+      file: 'dist/index.js',
+      sourcemap: false
+    },
+    plugins: [babel()]
+  },
+  {
+    input: 'src/index.js',
+    external: ['react', 'use-constant'],
+    output: {
+      format: 'cjs',
+      file: 'dist/index.cjs.js',
+      sourcemap: false
+    },
+    plugins: [babel()]
+  }
+]

